lib/models/blueprint.js

```diff
@@ -537,7 +537,7 @@ let Blueprint = CoreObject.extend({
       // Strip template tags
       const templateTagIdentifier = (index) => `template = __TEMPLATE_TAG_${index}__;`;
       const templateTagIdentifierBraces = (index) => `(template = __TEMPLATE_TAG_${index}__);`;
-      const templateTagMatches = preprocessor.parse(code);
+      const { code: templateTagMatches } = preprocessor.parse(code);
```

Comment:
We're using `parse` here, not `process`, so I think this change is not needed.

Reply:
@bertdeblock thank you @bertdeblock for pointing out! turns out I was not paying an attention

Reply:
No worries, I didn't notice it as well :smile: